tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -1307,3 +1308,164 @@ def test_update_record_goes_through_api_version_of_store_record_connection_timeo
+    side_effect=fake_download_file,
+)
+@mock.patch(
+    "inspirehep.modules.workflows.tasks.beard.json_api_request",
+    side_effect=fake_beard_api_request,
+)
+@mock.patch(
+    "inspirehep.modules.workflows.tasks.magpie.json_api_request",
+    side_effect=fake_magpie_api_request,
+)
+@mock.patch(
+    "inspirehep.modules.workflows.tasks.refextract.extract_references_from_file",
+    return_value=[],
+)
+@mock.patch(
+    "inspirehep.modules.workflows.tasks.actions._is_auto_rejected", return_value=True,
+)
+def test_workflow_checks_affiliations_if_record_is_not_important(
+    mocked_is_auto_rejected,
+    mocked_refextract_extract_refs,
+    mocked_api_request_magpie,
+    mocked_beard_api,
+    mocked_actions_download,
+    mocked_is_pdf_link,
+    mocked_arxiv_download,
+    workflow_app,
+    mocked_external_services,
+):
+    """Test a full harvesting workflow."""
+    record = generate_record()
+    record['authors'][0]['raw_affiliations'] = [{"value": "IN2P3"}, {"value": "Cern"}]
+    record['authors'][1]['raw_affiliations'] = [{"value": "Fermilab"}]
+    workflow_id = build_workflow(record).id
+    with patch.dict(workflow_app.config, {
+        'FEATURE_FLAG_ENABLE_REST_RECORD_MANAGEMENT': True,
+        'INSPIREHEP_URL': "http://web:8000"
+    }):
+        start("article", object_id=workflow_id)
+
+    collections_in_record = mocked_external_services.request_history[0].json()['_collections']
+    assert "CDS Hidden" in collections_in_record
+    assert "HAL Hidden" in collections_in_record
+    assert "Fermilab" in collections_in_record
+    assert "Literature" not in collections_in_record
+
+
+@mock.patch(
+    "inspirehep.modules.workflows.tasks.arxiv.download_file_to_workflow",
+    side_effect=fake_download_file,
+)
+@mock.patch("inspirehep.modules.workflows.tasks.arxiv.is_pdf_link")
+@mock.patch(
+    "inspirehep.modules.workflows.tasks.actions.download_file_to_workflow",
+    side_effect=fake_download_file,
+)
+@mock.patch(
+    "inspirehep.modules.workflows.tasks.beard.json_api_request",
+    side_effect=fake_beard_api_request,
+)
+@mock.patch(
+    "inspirehep.modules.workflows.tasks.magpie.json_api_request",
+    side_effect=fake_magpie_api_request,
+)
+@mock.patch(
+    "inspirehep.modules.workflows.tasks.refextract.extract_references_from_file",
+    return_value=[],
+)
+def test_workflow_do_not_changes_to_hidden_if_record_authors_do_not_have_interesting_affiliations(
+    mocked_refextract_extract_refs,
+    mocked_api_request_magpie,
+    mocked_beard_api,
+    mocked_actions_download,
+    mocked_is_pdf_link,
+    mocked_arxiv_download,
+    workflow_app,
+    mocked_external_services,
+):
+    """Test a full harvesting workflow."""
+    record = generate_record()
+    workflow_id = build_workflow(record).id
+    with patch.dict(workflow_app.config, {
+        'FEATURE_FLAG_ENABLE_REST_RECORD_MANAGEMENT': True,
+        'INSPIREHEP_URL': "http://web:8000"
```

Comment:
shouldn't this be part of the conftest config?

Reply:
We don't want to set this for all tests we have in there...